whitesource.config

```diff
@@ -0,0 +1,256 @@
+####################################################################
```

Comment:
what do you think about making this config a dot file or even better imo move the config into _hacking_. I really would like to reduce the clutter in the root directory.

Reply:
This is the default config provided by whitesource which is loaded from the root folder by the container which runs in the pipeline.

Reply:
and the location is not configurable?

Reply:
Will there be a change to the Prow config as well? Do you want this as a required status check? Will this even run on our prod cluster?

Reply:
The PR which changes the dev environment is linked in the description. 

https://github.com/jenkins-x/environment-tekton-weasel-dev/pull/405

Reply:
I guess that I can make the config path configurable  and when is not set can default to `.whitesource.config`

Reply:
> I guess that I can make the config path configurable and when is not set can default to .whitesource.config

that would be awesome

Reply:
I  changed the whitesource image to make to be able to setup the configuration path.

Reply:
Please could you also review the prow pull request? I think this can be merged first.